docs/operator/configuring-eventing-cr.md

```diff
@@ -0,0 +1,117 @@
+
+Suppose you use the custom tag v0.13.0 for all your images, and all the image links you have are able to accessible
+without secrets, and defined in the expected format: `docker.io/knative-images/${NAME}:v0.13.0`, you need to define your
+operator CR with following content:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  registry:
+    default: docker.io/knative-images/${NAME}:v0.13.0
+```
+
+With that said, you need to make sure your images are saved in the following link:
+
+* Image of `eventing-controller`: `docker.io/knative-images/eventing-controller:v0.13.0`.
+* Image of `eventing-webhook`: `docker.io/knative-images/eventing-webhook:v0.13.0`.
+* Image of `imc-controller`: `docker.io/knative-images/imc-controller:v0.13.0`.
+* Image of `imc-dispatcher`: `docker.io/knative-images/imc-dispatcher:v0.13.0`.
+* Image of `broker-controller`: `docker.io/knative-images/broker-controller:v0.13.0`.
+
+This is the simplest case to define your custom image links. Be aware that if you use this approach, you must make sure
+that all the images in Knative Eventing will be replaced with your own images.
+
+## Example 2 — download images from different repositories without secrets:
+
+If all the images are not saved in a uniformed format, we need to define them on a one-on-one basis.
+
+Suppose we would like to download the images as below:
+
+* Image of `eventing-controller`: `docker.io/knative-images-repo1/eventing-controller:v0.13.0`.
+* Image of `eventing-webhook`: `docker.io/knative-images-repo2/eventing-webhook:v0.13.0`.
+* Image of `imc-controller`: `docker.io/knative-images-repo3/imc-controller:v0.13.0`.
+* Image of `imc-dispatcher`: `docker.io/knative-images-repo4/imc-dispatcher:v0.13.0`.
+* Image of `broker-controller`: `docker.io/knative-images-repo5/broker-controller:v0.13.0`.
+
+Our operator CR needs to be revised as bellow:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  registry:
+    override:
+      eventing-controller: docker.io/knative-images-repo1/eventing-controller:v0.13.0
+      eventing-webhook: docker.io/knative-images-repo2/eventing-webhook:v0.13.0
+      imc-controller: docker.io/knative-images-repo3/imc-controller:v0.13.0
+      imc-dispatcher:: docker.io/knative-images-repo4/imc-dispatcher::v0.13.0
+      broker-controller: docker.io/knative-images-repo5/broker-controller:v0.13.0
+```
+
+This is how we define the image links mapping to the container or image name on a one-on-one basis. Be aware that you do
+not need to replace all images for deployments in this approach. You only need to define the ones you need to replace.
+
+## Example 3 — download images with secrets:
+
+No matter you use the key `default` or `override` to define where to download your images, when you need to access your
```

Comment:
Instead:

"If you use the `default` or `override` attributes to define image links, and the image links require private secrets for access, you must append the `imagePullSecrets` attribute."

I don't think you need to mention `spec.registry` since they will see this placement in the example.